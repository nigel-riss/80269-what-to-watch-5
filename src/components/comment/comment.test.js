import React from 'react';
import renderer from 'react-test-renderer';
import Comment from './comment.jsx';


it(`Should Comment render correctly`, () => {
  const tree = renderer
    .create(
        <Comment
          date="2020-11-03T13:38:44.769Z"
          name="John Doe"
          rating={8.5}
          text="This is mock comment"
        />
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
