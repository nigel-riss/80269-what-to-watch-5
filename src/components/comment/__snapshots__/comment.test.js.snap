// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should Comment render correctly 1`] = `
<div
  className="review"
>
  <blockquote
    className="review__quote"
  >
    <p
      className="review__text"
    >
      This is mock comment
    </p>
    <footer
      className="review__details"
    >
      <cite
        className="review__author"
      >
        John Doe
      </cite>
      <time
        className="review__date"
        dateTime="2020-11-03T13:38:44.769Z"
      >
        November 03, 2020
      </time>
    </footer>
  </blockquote>
  <div
    className="review__rating"
  >
    8,5
  </div>
</div>
`;
